client/components/OrderList.js

```diff
@@ -28,9 +30,23 @@ const OrderList = ({ orders }) => {
                 <td>
                   <CustomTooltip index={index} mongoId={order.id} />{" "}
                 </td>
+                <td>{users.find((user) => user.id === order.userId).name}</td>
                 <td>{order.createdAt.substring(0, 10)}</td>
+                <td>{order.createdAt.substring(11, 16)}</td>
```

Comment:
need to find a way to change the timezone from `UTC+0` to `UTC+7` bangkok, hanoi, jarkata in the future.